datascience/tables.py

```diff
@@ -5719,12 +5719,48 @@ def asdict(self):
             return collections.OrderedDict(zip(self._table.labels, self))
 
     class Rows(collections.abc.Sequence):
-        """An iterable view over the rows in a table."""
+        """
+        An iterable row-styled view of the table object that is passed as an argument.
+
+        args:
+        table- accepts a table object of class Table()
+
+        >>>  t = td.Table().with_columns({
```

Comment:
I'm unsure why we're not instantiating the `Table` class directly here. What's the `td` object here? I don't see it anywhere in the docs?

Reply:
That was a mistake on my side. I didn't add the proper initialization for Table() class. I have updated it now.